include/kernel.h

```diff
@@ -2049,12 +2049,12 @@ struct k_fifo {
 /**
  * @cond INTERNAL_HIDDEN
  */
-#define _K_FIFO_INITIALIZER(obj) \
+#define Z_FIFO_INITIALIZER(obj) \
```

Comment:
Same here. The object operated on is called k_fifo, so the name should contain K_FIFO